### PR TITLE
Handlers for Guild CheckIn, UpdateLeader, UpdateNotice, UpdateEmblem

### PR DESCRIPTION
**Pull request by kOchirasu**
Emblem doesn't actually work because idk what UGC changes are needed.

### CODE REVIEW

**Review by AngeloTadeucci — APPROVED** [review]

File: Maple2.Server.Game/Manager/ExperienceManager.cs [hunk 1]
```diff
@@ -103,6 +103,7 @@ public sealed class ExperienceManager {
         expGained += GetRestExp(expGained);
         LevelUp();
         AddPrestigeExp(message.Type());
+        // TODO: Achievement 22400001, 22400002
```
Comment (AngeloTadeucci) [inline Maple2.Server.Game/Manager/ExperienceManager.cs:106]: isnt just the case of using `ConditionUpdate` ?

File: Maple2.Server.Game/PacketHandlers/GuildHandler.cs [hunk 2]
```diff
@@ -457,20 +456,133 @@ private enum Command : byte {
...
+            GuildResponse response = World.Guild(request);
+            var error = (GuildError) response.Error;
+            if (error != GuildError.none) {
+                session.Send(GuildPacket.Error(error));
+                return;
+            }
 
+            session.Send(GuildPacket.CheckedIn());
+            session.Exp.AddExp(ExpType.guildUserExp, session.Guild.Properties.CheckInPlayerExpRate);
+            Item? guildCoin = session.Field.ItemDrop.CreateItem(30000861, 4, session.Guild.Properties.CheckInCoin);
```
Comment (AngeloTadeucci) [inline Maple2.Server.Game/PacketHandlers/GuildHandler.cs:491]: add the id to constants

**Review by AngeloTadeucci — CHANGES_REQUESTED** [review]

File: Maple2.Server.Game/Manager/ExperienceManager.cs (hunk 1 above)
Comment (kOchirasu) [reply Maple2.Server.Game/Manager/ExperienceManager.cs]: added, please double check it though since idk what im doing

**Review by AngeloTadeucci — APPROVED** [review]
looks good @Zintixx pls review too

File: Maple2.Server.Game/Manager/ExperienceManager.cs (hunk 1 above)
Comment (Zintixx) [reply Maple2.Server.Game/Manager/ExperienceManager.cs]: it looks like there's neither any code or target for exp (judging based on available quests/achievements with this condition. Add these cases to return true in `ConditionUtil.CheckCode` and `CheckTarget` in the switches so we know they've been addressed on those ends.

**Review by Zintixx — APPROVED** [review]

**Pull request merged**